.github/workflows/ci.yaml

```diff
@@ -41,6 +41,8 @@ jobs:
         sudo snap install juju-wait --classic
```

Comment:
What was here before can probably be taken out.  I think the actions-operator above now installs everything you should need (except maybe tox.  can't remember about tox)